users/views.py

```diff
@@ -4,6 +4,23 @@
 from rest_framework import permissions
 from users.permissions import IsAdminUser, IsOwnerOrAdmin
 
+from rest_framework.decorators import api_view
+from rest_framework.response import Response
+from rest_framework.reverse import reverse
+
+
+@api_view(['GET'])
+def api_root(request, format=None):
+    if not request.user.is_authenticated:
+        return Response({
```

Comment:
Why do you return response with all users when user is not authenticated?

Reply:
Just to include users app (as users list default) to main page, like build-in rest api root does